config/crd/crd-doc-config.yaml

```diff
@@ -0,0 +1,8 @@
+processor:
+  ignoreTypes:
+    # - "List$"
+    # - "Status$"
+  ignoreFields:
+    # - "status$"
```

Comment:
Is the whole `processor` section necessary, if everything inside is commented out?

Reply:
This is not necessary.
I've left it intentionally, as I'm not very familiar with the best practices of generating CRD docs - on what's worth keeping and what's not. The default and initial version had the status field ignored, but then the latest modifications around #653 would not be visible. 

How about a more descriptive comment https://github.com/kyma-project/kyma-infrastructure-manager/pull/1084/commits/42ff57ca91025eea4a596bf07d772bf1dec22bbd ?